docs/blog/2018-12-17-ibm-case-study/index.md

```diff
@@ -0,0 +1,59 @@
+
+The original site was a home-grown static site based on Markdown and React that was complicated to update. Add to that many, many hands and minds working on it constantly - some with technical experience and some without. The large volume of content and content creators made it difficult to accomplish the main business objective for the website, which was to make it easy for anyone to contribute content, regardless of their level of technical ability.
+
+The main goals for the new site were simple:
+
+- A faster and easier site that anyone could update - even if they didn’t know how to code
+- Good documentation - if anyone leaves the project, all the documentation is there, which eliminates the need to search around for how it was created
+
+## Easy onboarding with Gatsby
+
+Alison Joseph, Front-End Developer at IBM, took over this project from a coworker who had already done great work choosing the framework for the website rebuild. Taking on a new project mid-stream can be daunting but Alison found the Gatsby documentation was top notch and better than any other open source docs she’s seen.
+
+Alison was the only developer able to focus full-time on the Gatsby project and it took about six months before the new site launched in October 2018. She’s quick to point out if they had gone the custom, in-house route, the project would have easily taken twice as long and required more than one developer.
+
+<pullquote>
+Gatsby just works. With the old system, I had to fight with something for a few days to troubleshoot issues. With Gatsby, you install and it just works. It’s stress-free.
+</pullquote>
+
+With non-technical content writers and designers needing the ability to add content quickly to the site, the “second to none onboarding” made it easy for everyone, regardless of skill level, to be successful. The Gatsby documentation made it easy to onboard new developers and the process was simple enough for any of the content creators to understand. This achieved one of the most important goals for the website: the documentation and process was there for the next person so they wouldn’t struggle.
+
+<pullquote>
+A home-grown system goes down when the people who created it leave. Gatsby doesn’t allow for that - it grows with whoever is at the controls then seamlessly passes on to the next person.
+</pullquote>
+
+## A New Era
+
+The new Carbon Design System website launched October 3, 2018, and is referenced by designers and developers as an open source project as well as internally at IBM. With a seamless design feel, most users didn’t “see” any changes, but they sure felt them with the lightning fast page load times. Alison pointed out their new Lighthouse performance score went from 7 to 96 (that’s a 89 point jump!) with PWA, Accessibility, Best Practices, and SEO scores in the high 90s - 100.
+
+## Lighthouse Performance Scores with Gatsby
```

Comment:
Seems a bit weird to have a section that's just an image and a pullquote? @lindaleebumblebee

Reply:
This relates to the section above....I want to show the lighthouse scores and thought it needed a title for the image but it really is part of the section above. Do you have a recommendation on how to make this part of the section above it but still have a description/title for the image?

Reply:
I'd suggest removing the header text and adding a caption to the image.

Reply:
Great- that works for me (@amberleyromo 's suggestion about the header image instead of header text).